homeassistant/components/deconz/climate.py

```diff
@@ -196,36 +194,36 @@ def hvac_mode(self) -> str:
 
         Need to be one of HVAC_MODE_*.
         """
-        return self._deconz_to_hvac_mode.get(
-            self._device.mode,
-            HVAC_MODE_HEAT if self._device.state_on else HVAC_MODE_OFF,
-        )
+        if self._device.mode in self.supported_hvac_modes:
+            return HVAC_MODE_TO_DECONZ[self._device.mode]
```

Comment:
This is incorrect. The property must return a Home Assistant climate hvac mode constant.

Reply:
Thank you @MartinHjelmare ! Incredible that you catch stuff like this. Do you even sleep? :)

Reply:
Resolved https://github.com/home-assistant/core/pull/70148